pilot/pkg/security/authz/builder/builder.go

```diff
@@ -23,25 +23,31 @@ import (
 	httppb "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
 	rbactcppb "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/rbac/v3"
 	"github.com/envoyproxy/go-control-plane/pkg/wellknown"
+	"github.com/hashicorp/go-multierror"
 
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/networking/plugin"
 	"istio.io/istio/pilot/pkg/networking/util"
 	authzmodel "istio.io/istio/pilot/pkg/security/authz/model"
 	"istio.io/istio/pilot/pkg/security/trustdomain"
 	"istio.io/istio/pkg/config/labels"
-	"istio.io/pkg/log"
 )
 
 var (
-	authzLog = log.RegisterScope("authorization", "Istio Authorization Policy", 0)
+	rbacPolicyMatchNever = &rbacpb.Policy{
```

Comment:
one thing to double check is that this is never accidentally mutated

Reply:
this should be covered by the builder_test.go in the `allow-none-out.yaml`

Reply:
What I mean is this object is now shared between multiple listeners, which may lead to unexpected results. We do this in a lot of places already, so I am not sure its a *bad* thing, but it is a potential risk. I opened https://github.com/istio/istio/issues/28947 to track possibly adding testing around this type of usage

Reply:
after looking through the code I don't see any places this could occur though

Reply:
Oh, I see what you mean. I think it should be fine here, we don't modify the generated config out of the authz plugin (and also inside the plugin), I can hard-code this if there are concerns.